src/v/redpanda/admin_server.h

```diff
@@ -206,7 +206,9 @@ class admin_server {
      */
     template<auth_level required_auth, bool peek_auth = false, typename F>
     void register_route_raw_async(
-      ss::httpd::path_description const& path, F handler) {
+      ss::httpd::path_description const& path,
+      F handler,
+      ss::sstring content_type = "json") {
```

Comment:
Is this content_type correct?  Should it be `zip`?

Reply:
No it should not be zip. Other admin api methods such as `get_manifest` return json content type. This change is so previous methods don't have to change either.

Reply:
Look at how I defined `get_debug_bundle`